hockeysdk/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,8 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="18" />
```

Comment:
This change here caused a bug in my app - I have WRITE_EXTERNAL_STORAGE in a manifest file in a dependency module included before Hockey SDK, and in the merged manifest ended up this version with the `maxSdkVersion` filter, making the permission not to be granted nor requestable.

Guessing this can be an issue for other people so this change might be relevant to mention in release notes.

My solution was to use `tools:node="replace"` manifest merging strategy in a manifest later in the dependency tree:

     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" tools:node="replace"/>

Reply:
IMHO it is less invasive to use `tools:remove="android:maxSdkVersion"` instead of `tools:node="replace"` in such case.
Replace will cause that other attributes from low priority manifests will be replaced instead of being merged.
TL;DR
It seems that currently there only 2 attributes other than `maxSdkVersion` but it may change in the future.

Reply:
@koral-- awesome suggestion! I'll update our knowledgebase!